examples/dt/compute/amd-sev/edpm/openstackdataplanenodeset.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-amd-sev
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+  services:
+    - bootstrap
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - libvirt
+    - ovn
+    - neutron-sriov
+    - neutron-metadata
+    - nova-custom-sriov
+  preProvisioned: true
+  nodes:
+    edpm-computeamdsev-0:
+      ansible:
+        ansibleHost: 192.168.122.100 # CHANGEME
+      hostName: edpm-computeamdsev-0
+      networks:
+      - defaultRoute: true
+        fixedIP: 192.168.122.100 # CHANGEME
+        name: CtlPlane
+        subnetName: subnet1
+      - name: InternalApi
+        subnetName: subnet1
+      - name: Storage
+        subnetName: subnet1
+      - name: Tenant
+        subnetName: subnet1
+    edpm-computeamdsev-1:
+      ansible:
+        ansibleHost: 192.168.122.101 # CHANGEME
+      hostName: edpm-computeamdsev-1
+      networks:
+      - defaultRoute: true
+        fixedIP: 192.168.122.101 # CHANGEME
+        name: CtlPlane
+        subnetName: subnet1
+      - name: InternalApi
+        subnetName: subnet1
+      - name: Storage
+        subnetName: subnet1
+      - name: Tenant
+        subnetName: subnet1
+  networkAttachments:
+    - ctlplane
+  nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+    managementNetwork: ctlplane
+    ansible:
+      ansibleUser: cloud-admin
+      ansiblePort: 22
+      ansibleVars:
+        # CHANGEME -- see https://access.redhat.com/solutions/253273
+        # edpm_bootstrap_command: |
+        #       subscription-manager register --username <subscription_manager_username> --password <subscription_manager_password>
+        #       podman login -u <registry_username> -p <registry_password> registry.redhat.io
+        service_net_map:
+          nova_api_network: internal_api
+          nova_libvirt_network: internal_api
+        # CHANGEME
+        timesync_ntp_servers:
+          - hostname: clock.redhat.com
+        # CPU pinning settings
+        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=16 intel_iommu=on iommu=pt isolcpus=4-23,28-47" # CHANGEME
+        edpm_tuned_profile: "cpu-partitioning" 
+        edpm_tuned_isolated_cores: "4-23,28-47" # CHANGEME
```

Comment:
This can be removed since that is more relevant to sr-iov

Reply:
done